bagelbids/mappings.py

```diff
@@ -11,4 +11,9 @@
     "dti": "nidm:DiffusionTensor",
     "asl": "nidm:ArterialSpinLabeling",
 }
-NEUROBAGEL = {"participant": "nb:ParticipantID", "session": "nb:SessionID"}
+NEUROBAGEL = {
+    "participant": "bg:ParticipantID",
+    "session": "bg:SessionID",
```

Comment:
good question, but this is a different variable. the `bg:SessionID` here is what we expect to find in the `IsAbout` `TermURL` attribute for a column that has been labeled as containing session_id information by the annotation tool. This is the signal for us to interpret the values in this column as session IDs during the parsing of the phenotypic value. I just checked and I realize that in our docs: https://www.neurobagel.org/documentation/dictionaries/#session-identifier I have used a different identifier there (from NCIT) - and although that other identifier is probably more fitting, I'll change the docs to match `bg:SessionID` for now because I want to keep the number of vocabularies more manageable.